lib/torrent.js

```diff
@@ -90,6 +90,7 @@ function Torrent (torrentId, opts) {
   this.pieces = []
   this._selections = []
   this._critical = []
+  this._getExtraAnnounceOpts = opts.getExtraAnnounceOpts
```

Comment:
I'd rather name this consistently with the `bittorrent-tracker` option. `getAnnounceOpts`